testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -84,6 +84,36 @@ object ServiceTest {
      */
     def withCassandra(enabled: Boolean): Setup
 
+    /**
+     * Enable Cassandra.
+     *
+     * If enabled, this will start an embedded Cassandra server before the tests start, and shut it down afterwards.
+     * It will also configure Lagom to use the embedded Cassandra server. Enabling Cassandra will also enable the
+     * cluster.
+     *
+     * @return A copy of this setup.
+     */
+    def withCassandra(): Setup = withCassandra(true)
```

Comment:
Duplicated with `with` method with a parameterless variant. No need to be so verbose. 
`withCassandra()`, `withJdbc()` and `withCluster()` are clear enough. We don't need to pass `true`. 

On the other hand, to disable it one have to call `withCassandra(false)`. Which will probably never be used because it's disabled by default. 

Should we consider deprecation?

Reply:
Totally agree. Thanks!